### ISSUE TITLE
Accessible Authentication 2.2.6 - Security and other concerns

### ISSUE THREAD

**Issue by KimberleeD** [w3c/wcag21#542]
It is our understanding that that “recalling” and “transcribing” in the proposed SC 2.2.6 is synonymous with multi-factor authentication, or “MFA.”  MFA is defined in the New York Department of Financial Services “NYDFS” cybersecurity regulation or as follows (see  http://dfs.ny.gov/legal/regulations/adoptions/dfsrf500txt.pdf): 

(f) Multi-Factor Authentication means authentication through verification of at least two of the following types of authentication factors: 
              (1) Knowledge factors, such as a password; or 
              (2) Possession factors, such as a token or text message on a mobile phone; or 
              (3) Inherence factors, such as a biometric characteristic. 

Based on the above, we would like the following questions/issues to be resolved.

Generally companies set security policies using one or more of these considerations:

1. Customer needs
2. Legal requirements
3. Best practices in the industry
 
To flesh out each consideration a bit more: 
1. In terms of customer needs, a company may be obligated by contract with their customers to provide a very high level of security. This can and does conflict with the proposed SC. It’s debatable whether this would fall within the exceptions. Is it the intent of the SC to include the above as an exception?
 
2. In terms of legal requirements, many companies need to take into account things like the General Data Protection Regulation from the EU (see https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) which appears to be very strict and is soon to come into effect. For example, If a company fails to protect customer’s data, they could be fined 4% of their gross revenue. In addition, there are other regulations such as those from the NYDFS. (see http://www.dfs.ny.gov/about/press/pr1609131.htm and https://www.evolverinc.com/cybersecurity/ny-dfs-cyber-regulation-2017/). There are apparently a number of other regulations as well.
 
Even though the SC does have an exception that addresses specific legal requirements, given the importance and prevalence of these requirements, is the exception is so large and so important that it invalidates the rule?
Except for when any of the following are true:

- Authentication process involves basic personal identification information to which the user has easy access, such as name, address, email address and identification or social security number.
- **This is not achievable due to legal requirements**.

 3. Regarding industry standards, MFA seems to be a “best practice” in the privacy/security industry. Even doing 10-15 minutes of online searching retrieves many documents in which Multi-factor Authentication (MFA) is strongly recommended and is widely viewed as a best practice. Given this, can or should the WG continue to propose an SC that conflicts with important privacy and security considerations?
 
Finally, this SC may be introducing an unintended consequence. As I understand it, this SC is intended to empower those with memory and other cognitive impairments and to make it easier to log into sites. These individuals may be more at risk to security breaches (being tricked by phishing scams, etc.,). Therefore by lowering security to banking, medical, and other private information, aren’t we putting those individuals in a **more** vulnerable position? Do we, as a standards body, intend that? I think our goal is to make it easier and safer to interact with the digital world.
 
Obviously there is value in making authentication more accessible. This SC as written may not achieve that goal, and may make those with memory or other cognitive impairments more susceptible to security breaches.

**Comment by awkawk** [comment]
It is an interesting question that I'd like to hear from the COGA TF about - is the intent to provide options (e.g. you can reset a password or login with recalled information OR using a mechanism that doesn't require recalling information), or is it more inline with the multi-factor authentication (e.g. resetting a password with two of the three types mentioned by Kim).

I suspect that it is the first, and if that is the case, then for any site that needs to use MFA it would make it very hard for a site to accomplish that.

**Comment by awkawk** [comment]
Official WG response: Thank you for your comment. This SC has been removed from the draft and will be able to be considered in a future revision.
